examples/managers/chirps.py

```diff
@@ -7,7 +7,7 @@
 import pathlib
 import xarray as xr
 from gridded_etl_tools.dataset_manager import DatasetManager
-from gridded_etl_tools.utils.extractor import FTPExtractor
+from gridded_etl_tools.utils.extractor import FTPExtractor, pool
```

Comment:
Directly importing a name like pool can create name conflicts, I would consider calling pool as `extractor.pool` in CHIRPS and in the tests

```
import gridded_etl_tools.utils.extractor as extractor
```